### ISSUE TITLE
Support for Gephi 0.9

### ISSUE THREAD

**Issue by jmuyskens** [romanseidl/SHPExporter#2]
This plugin is no longer compatible with the latest version of Gephi.

**Comment by romanseidl** [comment]
Yes I know. It is also very unlikely I will find time to fix that in the very near future.

**Comment by mbastian** [comment]
For reference, this is also tracked here: https://github.com/gephi/gephi-plugins/issues/257
